evidence/ermalkaleci/2024-06-21.md

```diff
@@ -0,0 +1,32 @@
+# Retention at Rank 1
+
+|                  |                                 |
+| ---------------- | ------------------------------- |
+| **Report Date**  | Date of submission (2024/06/21) |
+| **Submitted by** | Ermal Kaleci                    |
+
+
+## Member details
+
+- Matrix username: @ermalkaleci:matrix.org
+- Polkadot address: 14iKbZws1fjJ6TH27yoRq6KeeVNof83VmxUBN2W2udQVBe5o
+- Current rank: 1
+- Date of initial induction: 2024-02-17
+- Date of last report: NA
+- Area(s) of Expertise/Interest: Frame, Testing, Tooling
+
+
+## Reporting period
+
+- Start date: 2024-04-02
+- End date: 2024-06-21
+
+
+## Evidence
+
+| Areas of Contribution | Tasks                                                        | Links                                                        | Notes                                    |
+| --------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ---------------------------------------- |
+| Polkadot SDK          | Improve MessageQueue pallet | https://github.com/paritytech/polkadot-sdk/pull/3844 | Add the ability for MessageQueue to process enqueued messages on idle |
+| Tooling               | Chopsticks        | http://github.com/AcalaNetwork/chopsticks | Core contributer |
+| Tooling               | Subway            | http://github.com/AcalaNetwork/subway | Core contributer |
+| Testing               | End to End tests  | https://github.com/AcalaNetwork/e2e-tests | Core contributer |
```

Comment:
I think it would help to have more granularity here of what was done.

Reply:
Sorry, I can't go through all the commits one-by-one